Sources/Commands/Utilities/DescribedPackage.swift

```diff
@@ -28,12 +28,13 @@ struct DescribedPackage: Encodable {
     let defaultLocalization: String?
     let platforms: [DescribedPlatformRestriction]
     let products: [DescribedProduct]
+    let executables: [DescribedExecutable]
     let targets: [DescribedTarget]
     let cLanguageStandard: String?
     let cxxLanguageStandard: String?
     let swiftLanguagesVersions: [String]?
 
-    init(from package: Package) {
+    init(from package: Package, executables: [(package: String?, name: String)]) {
```

Comment:
Why not use the newly introduced nominal type instead of a structurally-typed tuple?

```suggestion
    init(from package: Package, executables: [DescribedExecutable]) {
```